test/framework/clusterctl/e2e_config.go

```diff
@@ -800,9 +800,9 @@ func (c *E2EConfig) GetVariable(varName string) string {
 	return value
 }
 
-// GetVariableBestEffort returns a variable from environment variables or from the e2e config file.
+// GetVariable returns a variable from environment variables or from the e2e config file.
 // If the variable cannot be found it returns an empty string and does not fail.
-func (c *E2EConfig) GetVariableBestEffort(varName string) string {
+func (c *E2EConfig) GetVariable(varName string) string {
```

Comment:
Thinking a little bit more to this, what we are doing is changing the behavior of GetVariable, and this can be surprising for people using the test framework

```suggestion
func (c *E2EConfig) GetVariableOrEmpty(varName string) string {
```

Will address this issue a get to a state where we have two func to get variables, `GetVariableOrEmpty` and `MustGetVariable`, each with a well defined semantic.

Reply:
Done